### ISSUE TITLE
Imagenet VID dataset download

### ISSUE THREAD

**Issue by tumitx** [msracver/Deep-Feature-Flow#37]
could anyone share the Imagenet VID dataset? the original webset http://vision.cs.unc.edu/ilsvrc2015/ui/vid  seems down.

**Comment by isalirezag** [comment]
It does not work...

**Comment by huangbiubiu** [comment]
@YuwenXiong 's link is down now, is there any way to download it now?

**Comment by huangbiubiu** [comment]
For those who want vid dataset: this link is available

http://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16.php

**Comment by leimao** [comment]
> For those who want vid dataset: this link is available
> 
> http://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16.php

Thanks for mentioning this. It seems that the video is from ILSVRC2015 instead of ILSVRC2017. 

The ILSVRC2017 video dataset might be obtained from:
https://bvisionweb1.cs.unc.edu/ILSVRC2017/download-videos-1p39.php

**Comment by Sanku-Yogesh** [comment]
hi, which dataset exactly we need to download because there are huge, could u please refer to this comment https://github.com/Scalsol/mega.pytorch/issues/73#issue-754884469

**Comment by RyanMarten** [comment]
It seems like these links are dead now. Does anyone else have a copy hosted?

**Comment by angelomenezes** [comment]
Just checked the links and all of them are still down. I'm on the same team of @RyanMarten here.

**Comment by adilsonmedronha** [comment]
was anyone able to download the video dataset?

**Comment by AdrienGuo** [comment]
I also found the link https://bvisionweb1.cs.unc.edu/ilsvrc2015/ILSVRC2015_VID.tar.gz is not accessible.
The following error message appears:

===
Forbidden
You don't have permission to access this resource.
Apache/2.4.41 (Ubuntu) Server at bvisionweb1.cs.unc.edu Port 443

===

Hope someone can provide an alternative link 🙂.

**Comment by g1910** [comment]
> For those who want vid dataset: this link is available
> 
> http://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16.php

Thanks for sharing this. How to download from this php webpage? I am not sure if this link is working anymore.

**Comment by amaharek** [comment]
the link is not accessible, have you found a workaround for that issue?

**Comment by amaharek** [comment]
has anyone has a workaround for that issue?

**Comment by kaigeli** [comment]
It seems like these links are dead now. Does anyone else have a copy hosted? Thanks

**Comment by Rownak** [comment]
Hi, Most of the links are either down or Forbidden. Does anyone has a copy of this dataset who can upload and share it? 
I just found the following dataset which is just a small portion of the original Imagenet Vid dataset.
https://xdshang.github.io/docs/imagenet-vidvrd.html 
Thanks in advance.

**Comment by HamidrezaBsh** [comment]
Hi. I have been searching for this data set for two weeks and still there is no working link to download. even the imageNet help desk is not answering. my question is how new papers that are being released  since 2022 are citing this dataset while it is almost disappeared from the web.

**Comment by Seena-02** [comment]
Same issue as @HamidrezaBsh, unable to find entire dataset.

Update:
[https://www.kaggle.com/competitions/imagenet-object-localization-challenge/data?select=ILSVRC](url)

**Comment by SartisticV** [comment]
There is this issue on the matter, and it seems it has the entire ImageNet-VID dataset on pan baidu:

https://github.com/Hon-Wong/PTSEFormer/issues/10

However, you need to have a chinese phone number to access these files. Is there someone who could throw this on a google drive instead and share the link please? :)

**Comment by CatalinCraciun2002** [comment]
The image Net link still works but you have to modify the final path for it to open.

I found the following link to be working: https://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16-desconly.php

**Comment by JoemanJ** [comment]
The above link does indeed load, but all download links in it are forbidden

**Comment by CatalinCraciun2002** [comment]
Yes, it seems now it doesn't work anymore but I found a torrent that contains the whole ImageNet Vid dataset, including the 2015 VID. I have uploaded it to my drive bellow:
https://drive.google.com/file/d/1WZpKAGI-SfogA2-W1t_QOhq6qJ1Uamca/view?usp=drive_link

**Comment by DHWEUDH1** [comment]
The link is dead. Does anyone happen to have a backup or another way to download this? I'd really appreciate the help!

**Comment by junl19760806-afk** [comment]
> The link is dead. Does anyone happen to have a backup or another way to download this? I'd really appreciate the help!
You can take the idea that CatalinCraciun2002 had given on Mar 12, 2024，I find it‘s OK.
